src/app/doctor/[id]/components/activity/activity.component.tsx

```diff
@@ -0,0 +1,52 @@
+import CardComponent from "@/components/card/card.component";
+
+import MingcuteMedalLine from "@/icon/MingcuteMedalLine";
+import MingcuteChatLine from "@/icon/MingcuteChatLine";
+
+import styles from "./activity.module.css";
+
+type Props = {
+  doctorName: string;
+  yearActivity?: number;
+  monthActivity?: number;
+  activeConsultNumber?: number;
+};
+
+export default function ActivityComponent({
+  doctorName,
+  yearActivity = 0,
+  monthActivity = 0,
+  activeConsultNumber = 0,
+}: Props) {
+  return (
+    (yearActivity != 0 || monthActivity != 0 || activeConsultNumber != 0) && (
+      <div className={styles.activity}>
+        <b>فعالیت ها</b>
```

Comment:
بهتره قواعد نگارشی رو رعایت کنید
مثلاً اینجا باید از فاصله‌ی میان‌واجی استفاده میکردید به جای Space

```suggestion
        <b>فعالیت‌ها</b>
```